pkg/pf/tests/tfgen_test.go

```diff
@@ -33,7 +33,7 @@ import (
 )
 
 func TestRenameResourceWithAliasInAugmentedProvider(t *testing.T) {
-    t.Parallel()
+	t.Parallel()
```

Comment:
Any idea how this was checked in originally? Do we not lint these files?

Reply:
Yup we skip these: https://github.com/pulumi/pulumi-terraform-bridge/blob/2a8b877dbe500f33980524c68855f0e732c3d6aa/.golangci.yml#L41